service-clients-gen/src/main/java/com/flipkart/poseidon/serviceclients/generator/ServiceGenerator.java

```diff
@@ -294,6 +295,14 @@ private JInvocation createRequest(ServiceIDL serviceIdl, JCodeModel jCodeModel,
             }
             block.decl(jCodeModel.ref("String"), "uri", invocation);
         }
+
+        if (endPoint.getCommandName() != null && !endPoint.getCommandName().isEmpty() && endPoint.isDynamicCommandName()) {
+            matcher = PARAMETERS_PATTERN.matcher(endPoint.getCommandName());
```

Comment:
I think @shrey-garg  suggested to use just a boolean and generate a static param name like (commandName) instead of reading the actual param name using pattern matcher.

Reply:
yep thats what I had suggested